test/JDBC/upgrade/17_5/schedule

```diff
@@ -212,6 +212,7 @@ dateadd_internal_df
 datediff_big
 datediff
 dateadd
+datename_tzoffset-before-16_10-or-17_6
```

Comment:
I think should add this into not just this schedule file, but also other schedule files as well.